### PR TITLE
Astro: Added Astro discovery service

### PR DESCRIPTION
**Pull request by gerrieg**
Signed-off-by: Gerhard Riegler <gerhard.riegler@gmail.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/discovery/AstroDiscoveryService.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        super(ImmutableSet.of(new ThingTypeUID(BINDING_ID, "-")), DISCOVER_TIMEOUT_SECONDS, false);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    protected void startScan() {
+        logger.debug("Starting Astro discovery scan");
+        String result = HTTP.sendHttpGetRequest("http://ip-api.com/json/?fields=lat,lon");
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/discovery/AstroDiscoveryService.java]: Please use HttpUtil class directly - a binding must not have any dependency to a "model" bundle.

File: addons/binding/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/discovery/AstroDiscoveryService.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            logger.info("Evaluated Astro geolocation: latitude: {}, longitude: {}", latitude, longitude);
+
+            ThingTypeUID sunType = new ThingTypeUID(BINDING_ID, SUN);
+            ThingTypeUID moonType = new ThingTypeUID(BINDING_ID, MOON);
+
+            ThingUID sunThing = new ThingUID(sunType, AUTO);
+            ThingUID moonThing = new ThingUID(moonType, AUTO);
+
+            String propGeolocation = String.format("%s,%s", latitude, longitude);
+            thingDiscovered(DiscoveryResultBuilder.create(sunThing).withLabel("Autodiscovered Sun")
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/discovery/AstroDiscoveryService.java]: I'd maybe simply call it "local Sun" and "local Moon"

**Comment by gerrieg** [comment]
requested changes done

**Comment by kaikreuzer** [comment]
Thanks!

**Pull request merged**